.github/workflows/npm-publish.yml

```
@@ -0,0 +1,148 @@
+            echo "Version $PACKAGE_VERSION does not exist. Publishing..."
+            echo "version_exists=false" >> $GITHUB_OUTPUT
+
+            # Publish with provenance and conditional access for scoped packages
+            ACCESS_FLAG=""
+            if [[ "$PACKAGE_NAME" == @*/* ]]; then ACCESS_FLAG="--access public"; fi
+            npm publish --provenance $ACCESS_FLAG
+            echo "published=true" >> $GITHUB_OUTPUT
+          fi
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+        env:
```

actionlint (1.7.7)

key "env" is duplicated in element of "steps" section. previously defined at line:105,col:9

(syntax-check)